File: API-EVOSYSTEMS/API.EVOSYSTEMS/Models/Employee.cs
namespace API.EVOSYSTEMS.Models
{
    public class Employee
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Photo { get; set; }
        public int Rg { get; set; }
        public Guid DepartmentId { get; set; }
        public Department Department { get; set; }
    }
}

File: EvoSystem.UI/src/app/models/employee.ts
import { Department } from "./department";

export interface EmployeeBody {
    id: string;
    name: string;
    photo: string;
    rg: number;
    departmentId: string;
}

export interface EmployeeDTO {
    id: string;
    name: string;
    photo: string;
    rg: number;
    departmentId: string;
    department: Department;
}